viashpy/testing.py

```diff
@@ -136,8 +137,13 @@ def viash_source_config_path(meta_config_path, meta_config):
         # meta_config is a parsed viash config, retreive the location of the source
         return Path(meta_config["info"]["config"])
     except KeyError:
-        # If .['info']['config'] is not defined, assume that the config is a source config
-        return meta_config_path
+        # viash >= 0.9 defines build_info instead of info
+        try:
+            return Path(meta_config["build_info"]["config"])
+        except KeyError:
+            # If .['info']['config'] or .['build_info']['config'] is not defined,
+            # assume that the config is a source config
+            return meta_config_path
```

Comment:
I think it's better to invert these checks, because with viash 0.9 somebody could add a `.info.config` in their config.

Reply:
Done!